README.md

```diff
@@ -6,25 +6,6 @@ This is a bulk email sending service that sends email at a particular time set b
 - [MySQL](https://dev.mysql.com/doc/) The world's most popular open source database
 - Web hosting eg: [Hostinger](https://www.hostinger.com/web-hosting), [Namecheap](https://www.namecheap.com/hosting/), [Domainking](https://clients.domainking.ng/store/web-hosting) or [XAMPP Server](https://www.apachefriends.org/download.html)
 
-## How to install
-- Clone the repository
-- `git clone https://github.com/xyluz/php_queue_e.git`
-- `cd php_queue_e`
-- Spin off your development server
-
-## Live test
-- Try out the service [here](https://lunikdata.com/email-queue/send-email)
-- View [logs](https://lunikdata.com/email-queue/admin)
-
-## Notes
-- Verry crude, functionalities could be extened
-
-## Images
-
-<p align="center">
-  <img src="screenshot.jpg" width="100%" alt="bulk email screenshot" title="image sample">
-</p>
-
-<p align="center">
-  <img src="screenshot2.jpg" width="100%" alt="bulk email screenshot" title="image sample">
-</p>
+## Note
+### Would we need a cron job to get the entire thing working?
+- The code here does not require a cron job to run, as it contains a main loop that continuously processes the email queue until it is empty.
```

Comment:
The while loop will only run if the page is open. A queue system should be better made to ensure continuous background work is going on. That's the entire point of queues in this case.